tests/integration-service/integration.go

```diff
@@ -98,6 +100,19 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 					return nil
 				}, timeout, interval).Should(Succeed())
 			})
+
+			It("verifies that the finalizer has been removed", func() {
+				timeout := "60s"
+				interval := "1s"
+				Eventually(func() error {
+					// This is broken because the function is just checking the object rather than going to the cluster
```

Comment:
@14rcole  Just out of curiosity, what does this comment mean?
IIUC the function `GetBuildPipelineRun()` does [make calls to the cluster](https://github.com/redhat-appstudio/e2e-tests/blob/6ee173b10a11c975c219f809882d7d9d1411f597/pkg/clients/integration/pipelineruns.go#L73) to fetch the build PLRs.